lib/mongoid/geospatial/wrappers/rgeo.rb

```diff
@@ -1,5 +1,5 @@
 require 'rgeo'
-require 'mongoid_geospatial/ext/rgeo_spherical_point_impl'
+require 'mongoid/geospatial/ext/rgeo_spherical_point_impl'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.